### DISCUSSION TITLE
Deploying Next.js on custom server & using <Image>

### DISCUSSION THREAD

**Discussion by testdgb** [vercel/next.js#49050] [Help]
### Summary

Hey there, 
I'm working on my website using next.js and I decided to self-deploy it on a VPS/hosting server.
My question is, I'm using the <Image> tag on my website, but I noticed that Vercel actually charges for the Image Optimization, so if I deploy the website on my own, what will happen to the images with the <Image> tag? Will they still be optimized for free? How will it work? 

Thanks!

**Comment by devjiwonchoi** [comment]
You've got mixed up on the [Image Optimization with Vercel](https://vercel.com/docs/concepts/image-optimization) and the [Image Optimization](https://nextjs.org/docs/pages/building-your-application/optimizing/images) feature of `next/image`. 
If you have hosted your website on your own server, you will not be charged.
